pkg/specgen/volumes.go

```diff
@@ -65,6 +67,20 @@ func GenVolumeMounts(volumeFlag []string) (map[string]spec.Mount, map[string]*Na
 			err     error
 		)
 
+		// Support relative paths beginning with ./
+		v := strings.Split(vol, ":")
+		if m1.MatchString(v[0]) {
```

Comment:
Regex is expensive, I think `strings.HasPrefix()` should be better here

Also why split this her and then below again? Just make the evalSymlink on the src below